server/util/redact/redact.go

```diff
@@ -211,6 +211,12 @@ func stripNonAllowedEnvVars(tokens []string) {
 	}
 }
 
+func redactNonAllowedEnvVars(txt string) string {
+	// Create a regex that matches env vars but stops at whitespace or common delimiters
+	envVarTextRegex := regexp.MustCompile(`(--(?:` + strings.Join(envVarOptionNames, "|") + `)=\w+=)[^\s#]*`)
```

Comment:
It seems safer to match wrap `envVarOptionNames` entries in `regexp.QuoteMeta`. We don't expect special chars in env var names, but they are not forbidden by any spec and could cause this line to panic.

Reply:
Good call, @fmeum! Fixed.